packages/utils/package.json

```diff
@@ -27,5 +27,8 @@
   "sideEffects": false,
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "@adobe/react-spectrum": "^3.29.0"
```

Comment:
This seems like a pretty large dependency to add to `@deephaven/utils` which currently has no other dependencies. There's basically two things added to utils that require this:
1. `SpectrumUtils`: That's only used within `@deephaven/components` and `SelectionUtils`, which is also used in `react-hooks`. Should probably move it in there.
2. `WindowedListData`: Used within jsapi-components, jsapi-utils, react-hooks. jsapi-components depends on jsapi-utils which depends on react-hooks, so move `WindowedListData`, so I think `react-hooks` would be a more appropriate place for that.

`@deephaven/utils` is really just for vanilla utility methods, and shouldn't have any large dependencies.

Reply:
Spectrum dependency removed from @deephaven/utils